packages/components/psammead-amp-geo/README.md

```diff
@@ -1,5 +1,7 @@
 # psammead-amp-geo - [![Known Vulnerabilities](https://snyk.io/test/github/bbc/psammead/badge.svg?targetFile=packages%2Fcomponents%2Fpsammead-amp-geo%2Fpackage.json)](https://snyk.io/test/github/bbc/psammead?targetFile=packages%2Fcomponents%2Fpsammead-amp-geo%2Fpackage.json) [![Dependency Status](https://david-dm.org/bbc/psammead.svg?path=packages/components/psammead-amp-geo)](https://david-dm.org/bbc/psammead?path=packages/components/psammead-amp-geo) [![peerDependencies Status](https://david-dm.org/bbc/psammead/peer-status.svg?path=packages/components/psammead-amp-geo)](https://david-dm.org/bbc/psammead?path=packages/components/psammead-amp-geo&type=peer) [![Storybook](https://raw.githubusercontent.com/storybooks/brand/master/badge/badge-storybook.svg?sanitize=true)](https://bbc.github.io/psammead/?path=/story/figure--containing-image) [![GitHub license](https://img.shields.io/badge/license-Apache%202.0-blue.svg)](https://github.com/bbc/psammead/blob/latest/LICENSE) [![npm version](https://img.shields.io/npm/v/@bbc/psammead-amp-geo.svg)](https://www.npmjs.com/package/@bbc/psammead-amp-geo) [![PRs Welcome](https://img.shields.io/badge/PRs-welcome-brightgreen.svg)](https://github.com/bbc/psammead/blob/latest/CONTRIBUTING.md)
 
+# Deprecation notice - this component is no longer being maintained.
```

Comment:
Do we need to use this utility: https://docs.npmjs.com/cli/v7/commands/npm-deprecate ? I don't if we've done this for other packages we've deprecated?

How have we deprecated other packages? I think we'd be better actually deleting the code as we'll end up maintaining unused code in the future if we make a change like move to `yarn` for example.

Reply:
This PR mimics this other one: https://github.com/bbc/psammead/pull/4425, couldn't find any documentation for deprecating a psammead package. Happy to use the npm utility and/or delete the code but the npm utility would need to be run by the owner of the package.

Reply:
I'll broaden this conversation a bit, I'm a bit concerned about us accidently maintaining dead code